### DISCUSSION TITLE
Hugo 极简主题 Echo - forecho's Blog

### DISCUSSION THREAD

**Discussion by giscus** [forecho/hugo-blog#23] [Announcements]
# Hugo 极简主题 Echo - forecho's Blog

引言
去年年初的时候博客从 Octopress 迁移到 Hugo，那个时候找遍了很多 Hugo 主题都不满意，于是自己写了一个。
这就是 echo 主题的由来。最近又优化了一下这个主题，就顺便写篇文章给大家介绍一下这个主题，刚写出来的时候只在 V2EX 上分享过。
特色
风格基于 Tailwind CSS
TailwindCSS 可以说是这两年比较火的 CSS 框架了，看腻了 Bootstrap 的样式，觉得 TailwindCSS 真漂亮，但是在使用上不同于 Bootstrap， TailwindCSS 给人的一种感觉是面向过程写代码，可以自己自由组装各种样式，缺点是学习门槛高一点。而且可能要写大量重复的代码。
这次采用 TailwindCSS 做为主题 CSS 框架，一是因为它样式好看，二是尝鲜，换思路写 CSS。

https://blog.forecho.com/hugo-theme-echo.html

**Comment by yydschen** [comment]
主题不错！值得收藏~
